social_distribution/author_manager/views.py

```diff
@@ -489,21 +492,21 @@ def post(self, request):
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 
-class LikeAPI(APIView):
```

Comment:
Is there anything wrong with this API? Why was this commented out?

Reply:
Ohh I decided to move it to Inbox API. but I forgot to do it here. Let me fix that! Thanks